tests/data/demo_sot_data/lasot/create_assets.py

```diff
@@ -43,7 +43,9 @@ def create_dummy_data():
                       int(h)],
                 area=int(w) * int(h),
                 full_occlusion=False,
-                out_of_view=False)
+                out_of_view=False,
+                iscrowd=False,
+                occluded=False)
```

Comment:
It is unnecessary to use the `lasot.json` in unittest, you can use the `coco_vid`.json.

Besides, you can use `getattr(dict, key, default_value)` to get some attribution rather than generate the attribution in the JSON file.